src/jaxsim/integrators/common.py

```diff
@@ -211,7 +211,7 @@ def build(
 
         # Adjust the shape of the tableau coefficients.
         c = jnp.atleast_1d(cls.c.squeeze())
-        b = jnp.atleast_2d(jnp.vstack(cls.b.squeeze()))
+        b = jnp.atleast_2d(cls.b.T.squeeze()).transpose()
```

Comment:
Probably is better that the check on the butcher tableau fails rather than monkey-patching the class attributes. I am going to remove this adjustment.